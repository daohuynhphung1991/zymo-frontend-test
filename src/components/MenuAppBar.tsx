import * as React from "react";
import AppBar from "@mui/material/AppBar";
import Box from "@mui/material/Box";
import Toolbar from "@mui/material/Toolbar";
import Typography from "@mui/material/Typography";
import logo from "../images/logo.png";

export default function MenuAppBar() {
  return (
    <Box sx={{ flexGrow: 1 }}>
      <AppBar position="static">
        <Toolbar>
          <img src={logo} alt={"corona virus"} width={"auto"} height={40} />
          <Typography variant="h6" component="div">
            <span style={{ marginLeft: 15 }}>{" Coronavirus disease"}</span>
          </Typography>
        </Toolbar>
      </AppBar>
    </Box>
  );
}
