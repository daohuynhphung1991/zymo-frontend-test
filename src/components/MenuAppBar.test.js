import React from "react";
import renderer from "react-test-renderer";
import MenuAppBar from "./MenuAppBar";

test("MenuAppBar Component", () => {
  const component = renderer.create(
    <MenuAppBar>This is Card Component</MenuAppBar>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
