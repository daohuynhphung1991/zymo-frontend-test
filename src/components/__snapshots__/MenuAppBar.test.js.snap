// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuAppBar Component 1`] = `
<div
  className="MuiBox-root css-i9gxme"
>
  <header
    className="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hip9hq-MuiPaper-root-MuiAppBar-root"
  >
    <div
      className="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-1t29gy6-MuiToolbar-root"
    >
      <img
        alt="corona virus"
        height={40}
        src="logo.png"
        width="auto"
      />
      <div
        className="MuiTypography-root MuiTypography-h6 css-2ulfj5-MuiTypography-root"
      >
        <span
          style={
            Object {
              "marginLeft": 15,
            }
          }
        >
           Coronavirus disease
        </span>
      </div>
    </div>
  </header>
</div>
`;
